From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ikey Doherty <michael.i.doherty@intel.com>
Date: Fri, 6 Mar 2015 14:57:27 +0000
Subject: [PATCH] Use modern defaults

Signed-off-by: Ikey Doherty <michael.i.doherty@intel.com>
---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 7aa729d..9d984e5 100644
--- a/Makefile
+++ b/Makefile
@@ -31,8 +31,8 @@ CFLAGS += -D_FILE_OFFSET_BITS=64
 LDFLAGS ?=
 
 DESTDIR =
-prefix  = /usr/local
-sbindir = $(prefix)/sbin
+prefix  = /usr/
+sbindir = $(prefix)/bin
 mandir  = $(prefix)/share/man
 man8dir = $(mandir)/man8
 docdir  = $(prefix)/share/doc/dmidecode
