From 50001a81337d39985c47bd3b4dcac0a2541855ef Mon Sep 17 00:00:00 2001
From: Ikey Doherty <michael.i.doherty@intel.com>
Date: Fri, 6 Mar 2015 14:57:27 +0000
Subject: [PATCH] Use modern defaults

Signed-off-by: Ikey Doherty <michael.i.doherty@intel.com>
---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 55378bb..f1a4c60 100644
--- a/Makefile
+++ b/Makefile
@@ -26,8 +26,8 @@ CFLAGS += -O2
 LDFLAGS =
 
 DESTDIR =
-prefix  = /usr/local
-sbindir = $(prefix)/sbin
+prefix  = /usr/
+sbindir = $(prefix)/bin
 mandir  = $(prefix)/share/man
 man8dir = $(mandir)/man8
 docdir  = $(prefix)/share/doc/dmidecode
-- 
1.9.1

